src/GregClient/Utility/FileUtilities.cs

```diff
@@ -167,13 +167,43 @@ public static string UnZip(string zipFilePath)
         /// </summary>
         /// <param name="zipFilePath"></param>
         /// <returns></returns>
-        public static void UnZip(string zipFilePath, string unzipDirectory)
+        public static void UnZip(string zipFilePath, string destinationDirectoryName)
```

Comment:
Good point -- I changed it to align with the implementation in System.IO.Compression.ZipFileExtensions.ExtractToDirectory, but that's completely unnecessary, I'll revert the name change to avoid the source level break.